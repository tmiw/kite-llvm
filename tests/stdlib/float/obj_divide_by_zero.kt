run [
    (6.0|obj / 0.0|obj)|print;
] catch [
    __exc.__name|print;
    __exc.message|print;
];
