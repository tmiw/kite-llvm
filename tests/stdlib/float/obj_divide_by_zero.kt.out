System.exceptions.DivideByZero
Division by zero attempted
